src/Libraries/RevitServices/Materials/Materials.cs

```diff
@@ -118,18 +116,14 @@ private void FindOrCreateDynamoErrorMaterial()
 
         private void FindDynamoGraphicsStyle()
         {
-            var styles = new FilteredElementCollector(DocumentManager.Instance.CurrentUIDocument.Document);
-            styles.OfClass(typeof(GraphicsStyle));
-
-            var gStyle = styles.ToElements().FirstOrDefault(x => x.Name == "Generic Models");
-
-            if (gStyle != null)
+            var genericModelCategory = Category.GetCategory(DocumentManager.Instance.CurrentDBDocument, BuiltInCategory.OST_GenericModel);
```

Comment:
Look-up anything built in by name is usually a bad idea. Will not work on localized templates/projects.